### PR TITLE
C2 mahmoud identify buyer

### PR DESCRIPTION
**Pull request by mahmoudHussein**
@maizaied @HebaElSobky @mohamedahmedawad93 please review and give feed back

### CODE REVIEW

**Comment by BeshoyAtef** [comment]
#104

**Comment by mahmoudHussein** [comment]
@maizaied  @mohamedahmedawad93 @HebaElSobky PLEASEEE MERGE YOU REVIEW ME !!

**Comment by mahmoudHussein** [comment]
@maizaied @mohamedahmedawad93 @HebaElSobky please review me and merge if correct.

File: project/tager/tager/urls.py~ [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+urlpatterns += patterns('fbregister.facebook',
+	url(r'^facebook/login/$', 'facebook_login', name="facebook_login"),
+    url(r'^facebook/login/done/$', 'facebook_login_done', name="facebook_login_done"),
+)
+
+urlpatterns += patterns('fbregister.views',
+    url(r'^$', 'index', name="index"),
+)
```
Comment (BeshoyAtef) [inline project/tager/tager/urls.py~:40]: this file is not needed !!

**Comment by HebaElSobky** [comment]
@mahmoudHussein same error as @maizaied  error during template rendering w global name 'User' not defined please check it

**Comment by HebaElSobky** [comment]
@mahmoudHussein your scenario has been reviewd :+1:

**Pull request merged**